client/platform/desktop/backend/serializers/viame.ts

```diff
@@ -300,7 +300,7 @@ async function writeHeader(writer: Writable, meta: JsonMeta) {
     writer.write([`#meta fps=${meta.fps}`]);
   }
   writer.write(
-    [`# Written on ${(new Date()).toLocaleString()} by dive_csv_writer:typescript`],
+    [`# Written on ${(new Date()).toLocaleString().replace(',', '')} by dive_writer:typescript`],
```

Comment:
localeString was returning a string with a comma, causing it to get quoted, which is bad.